sdk/highlight-py/e2e/highlight_flask/app.py

```diff
@@ -12,10 +12,7 @@
 app = Flask(__name__)
 H = highlight_io.H(
     "1",
-    integrations=[
-        FlaskIntegration(),
-        RequestsIntegration(tracing_origins=["localhost", "highlight.io"]),
-    ],
+    integrations=[FlaskIntegration()],
```

Comment:
don't we still need to have `RequestsIntegration()` here to have that instrumentation enabled?

Reply:
Nope! This is instrumented by default if the appropriate libraries are installed. We just had this here before to define the `tracing_origins`, which are no longer used